pom.xml

```diff
@@ -99,6 +99,9 @@
             </googleJavaFormat>
             <trimTrailingWhitespace/>
             <removeUnusedImports/>
+<!--            <licenseHeader>-->
```

Comment:
What's this?

Reply:
Ah yes that snuck in--I was playing around with the idea of adding a header to all of our source files to indicate something about them being from Code4Community. We can do this relatively easily through spotless (I think) but I didn't commit to it